posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -236,6 +251,13 @@ public class PostHog private constructor(
             }
         }
 
+        synchronized(sessionLock) {
+            if (sessionId != sessionIdNone) {
+                props["\$session_id"] = sessionId.toString()
+                props["\$window_id"] = sessionId.toString()
```

Comment:
@pauldambra do you know if we still need this? should I generate a new UUID or can I reuse the `session_id` for the `window_id`?